src/main/java/com/squadAlertSystem/squadalertsystem/dto/response/GetMemberResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.squadAlertSystem.squadalertsystem.dto.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class GetMemberResponse {
+    private String id;
+    private String name;
+    private String email;
+    private String phone;
```

Comment:
You can simply return Member Entity in response instead of creating identical GetMemberResponse here. Both look the same.